### ISSUE TITLE
Cannot import existing key

### ISSUE THREAD

**Issue by jahlives** [roundcube/roundcubemail#5276]
I just installed a RC 1.2.0 test on my webserver. configured everything to enable enigma plugin. Then tried to import an existing key. It loads for quite a while and then stop without any error message in the webinterface. Under the PGP Key tab no key is shown. Generating new keys work but not importing exsting ones (at least in my case)
The webserver process has write access to enigma/home directory as the files secring.gpg and puring.gpg do exists after importing. They're just empty
Any idea whats going wrong? could it be related to the fact that my Apache Server runs with suexec?
Following the logs from enigma plugin for importing an existing key

> [25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --always-trust --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --version
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: BEGIN PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG output stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG output
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 652 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 3
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG output stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG output
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG status stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG status
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: END PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: CLOSING GPG SUBPROCESS
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
> [25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-secret-keys
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: BEGIN PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 5 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 131 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: ERROR: gpg: keyring `/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch/secring.gpg' created
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 136 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: ERROR: gpg: keyring `/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch/pubring.gpg' created
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 3
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG output stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG output
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG status stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG status
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: END PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: CLOSING GPG SUBPROCESS
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-public-keys
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: BEGIN PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 1
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG output stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG output
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 18 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: selecting streams
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => got 3
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG output stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG output
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG error stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG error
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: GPG status stream ready for reading
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => about to read 8192 bytes from GPG status
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: => read 0 bytes
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: END PROCESSING
[25-May-2016 10:59:07 +0200]: <7nd2n6c3> GPG: CLOSING GPG SUBPROCESS

**Comment by jahlives** [comment]
Half "solved" by importing the keypair manually on cli with gpg. Now can send encrypted mail, BUT if I choose to sign the encrypted mail the mail is not sent without any error. RC just stays in mail creation window.

**Comment by alecpl** [comment]
Provide the debug log from the moment when you import a key or when you send the email This looks like a problem with PinEntry. Check for errors in Roundcube log directory but also in Apache log.

**Comment by jahlives** [comment]
Sorry for the stupid question :-) But is the debug log more than just setting
> $config['enigma_debug'] = true;

in plugin/enigma/config.inc.php ?

**Comment by alecpl** [comment]
This is it. You already provided some debug, but I think not from the moment of the issue occurred.

**Comment by jahlives** [comment]
When I click on SEND for a message to be signed, thats exaclty what I get from enigma debug

[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --always-trust --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --version
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 652 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 3
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-secret-keys 'tobster@brain-force.ch'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 617 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 3
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-public-keys 'tobster@brain-force.ch'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 636 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 3
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-keys '351CE625A868A4E4'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 517 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 2
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --armor --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --export '1243F0CC6149D675549DF3F6351CE625A868A4E4'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 6362 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 2
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-secret-keys 'tobster@brain-force.ch'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 617 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 2
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: USING GPG 1.4.18 with PHP 5.6.20-0+deb8u1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: OPENING GPG SUBPROCESS WITH THE FOLLOWING COMMAND:
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: /usr/bin/gpg --status-fd '3' --command-fd '4' --no-secmem-warning --no-tty --no-default-keyring --no-options --no-use-agent --no-permission-warning --exit-on-status-write-error --trust-model always --with-colons --with-fingerprint --with-fingerprint --fixed-list-mode --homedir '/var/www/brain-force.ch/htdocs/webmail-new.brain-force.ch/plugins/enigma/home/tobster@brain-force.ch' --list-public-keys 'tobster@brain-force.ch'
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: BEGIN PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 636 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 1
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG status stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG status
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: selecting streams
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => got 2
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG output stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG output
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: GPG error stream ready for reading
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => about to read 8192 bytes from GPG error
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: => read 0 bytes
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: END PROCESSING
[26-May-2016 11:29:45 +0200]: <fm2klj7t> GPG: CLOSING GPG SUBPROCESS

**Comment by alecpl** [comment]
The log is fine. When sending a signed email you should be asked for a passphrase (until the password is stored in session, so please make sure to re-login before testing). So, looks like problem with PinEntry. Did you check apache error log?

**Comment by jahlives** [comment]
Apache error log does not show one additional line when trying to send the mail. I'm not asked for a password as my key is not password protected (I know I should change that ;-) )

**Comment by alecpl** [comment]
That might be the problem. I never tested a key with no password.

**Comment by alecpl** [comment]
I confirmed that keys with no password aren't supported, but anyway in that case Roundcube still should display a password entry dialog. So, there's something else going on here.

**Comment by jahlives** [comment]
I added a password via gpg command on cli but still does not show any password dialog. Could the issue be related to the fact that my apache runs with suexec?

**Comment by alecpl** [comment]
Do you see any javascript errors in browser console? What browser? Try with a different browser. Check http request response.

**Comment by jahlives** [comment]
I normally use Google Chrome on Fedora 23. Tried with Firefox. Same issue there. The console says

> TypeError: c.stopPropagation is not a function
rcube_webmail.prototype.enigma_password_request/<.click()
 enigma.min.js:20
n.access()
 jquery.min.js:34
n.access()
 jquery.min.js:34
.attr()
 jquery.min.js:35
._createButtons/<()
 jquery-ui-1.9.1.custom.min.js:6
.each()
 jquery.min.js:34
._createButtons()
 jquery-ui-1.9.1.custom.min.js:6
e.widget/</r[t]</<()
 jquery-ui-1.9.1.custom.min.js:6
._create()
 jquery-ui-1.9.1.custom.min.js:6
e.widget/</r[t]</<()
 jquery-ui-1.9.1.custom.min.js:6
e.Widget.prototype._createWidget()
 jquery-ui-1.9.1.custom.min.js:6
e.widget/e[a][t]()
 jquery-ui-1.9.1.custom.min.js:6
e.widget.bridge/e.fn[n]/<()
 jquery-ui-1.9.1.custom.min.js:6
.each()
 jquery.min.js:34
n.prototype.each()
 jquery.min.js:34
e.widget.bridge/e.fn[n]()
 jquery-ui-1.9.1.custom.min.js:6
rcube_webmail/this.show_popup_dialog()
 app.min.js:274
rcube_webmail.prototype.enigma_password_request()
 enigma.min.js:20
<anonym>
 webmail.brain-force.ch:9

**Comment by alecpl** [comment]
Ok, so we got something. We do not use jquery-UI-1.9 in Roundcube 1.2, something is wrong with your installation. It looks like jqueryui plugin is in some old version.

**Comment by jahlives** [comment]
@alecpl 
Thanks a lot, you gave me the right hint :-) I forgot that after install the new roundcube I just copied the plugins folder from my old installation. Now I removed the jqueryui folder, copied the one from the recent RC version and the password window appears. 
After adding the password to the key again the mail has been sent. Just with no password does not work, but I can live with that :-)
Now I will delete the key and see if the problem with importing the key was also jqueryui related
